modules/vectra.py

```diff
@@ -65,6 +68,17 @@ def __init__(self, response):
         super().__init__(response)
 
 
+class CustomException(HTTPException):
+    "Custom Exception raised while failure occurs."
+
+    pass
+
+
+def kill_process_and_exit():
+    logging.error("Error obtaining access token. Exiting.")
```

Comment:
The name of this function is generic, and not specific to token issues.  Should the message string for the log be passed in?